<ui version="4.0" >
 <class>QmitkImageCropperControls</class>
 <widget class="QWidget" name="QmitkImageCropperControls" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>651</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>BoundingObjectImageCropper</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Select the image to crop:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkDataStorageComboBox" name="cmbImage" />
   </item>
   <item>
    <widget class="QCheckBox" name="m_EnableSurroundingCheckBox" >
     <property name="text" >
      <string>Set border voxels to specific value after cropping</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_TLGrayvalue" >
     <property name="text" >
      <string>Select grayvalue of the border voxels</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QSlider" name="m_SurroundingSlider" >
       <property name="minimum" >
        <number>-2000</number>
       </property>
       <property name="maximum" >
        <number>2000</number>
       </property>
       <property name="value" >
        <number>1000</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_SurroundingSpin" >
       <property name="minimum" >
        <number>-32767</number>
       </property>
       <property name="maximum" >
        <number>32767</number>
       </property>
       <property name="value" >
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="btnCrop" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>crop the selected image</string>
     </property>
     <property name="text" >
      <string>Crop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_NewBoxButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>crop the selected image</string>
     </property>
     <property name="text" >
      <string>New bounding box!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkInformation" >
     <property name="text" >
      <string>Show usage information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupInfo" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="lblInfo" >
        <property name="text" >
         <string>The yellow box around the selected image can be changed in size and orientation. You can use it so mark an interesing region of the image and then use the "Crop" button to cut off parts of the image that are outside the box.&lt;br>&#xd;&#xd;
The original image will not be modified, it will only be hidden.</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="pixlblWhatToDo" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap>:/imagecropper/btn_ctrl.xpm</pixmap>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pixlblWhatToDo_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap>:/imagecropper/mouse_left.xpm</pixmap>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>Move the box</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="pixlblWhatToDo_3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="pixmap" >
           <pixmap>:/imagecropper/btn_ctrl.xpm</pixmap>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pixlblWhatToDo_2_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap>:/imagecropper/mouse_middle.xpm</pixmap>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2_2" >
          <property name="text" >
           <string>Rotate the box</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="pixlblWhatToDo_3_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="pixmap" >
           <pixmap>:/imagecropper/btn_ctrl.xpm</pixmap>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pixlblWhatToDo_2_2_2" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap" >
           <pixmap>:/imagecropper/mouse_right.xpm</pixmap>
          </property>
          <property name="scaledContents" >
           <bool>false</bool>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2_2_2" >
          <property name="text" >
           <string>Resize the box</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>200</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >mitkDataTree.h</include>
  <include location="local" >QmitkDataStorageComboBox.h</include>
 </includes>
 <resources>
  <include location="../../resources/imagecropper.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_SurroundingSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_SurroundingSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_SurroundingSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_SurroundingSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
